docs/_spec/06-expressions.md

```diff
@@ -455,6 +459,7 @@ The final expression can be omitted, in which case the unit value `()` is assume
 The expected type of the final expression ´e´ is the expected type of the block.
 The expected type of all preceding statements is undefined.
 
+<!-- TODO: Rewrite when type avoidance section is done -->
 The type of a block `´s_1´; ...; ´s_n´; ´e´` is `´T´ forSome {´\,Q\,´}`, where ´T´ is the type of ´e´ and ´Q´ contains [existential clauses](03-types.html#existential-types) for every value or type name which is free in ´T´ and which is defined locally in one of the statements ´s_1, ..., s_n´.
```

Comment:
This sentence contains a broken link to the old existential types section, and expands the type of a block as a type that uses `forSome`. I believe that should be addressed before we merge the PR.

Reply:
"Fixed" by under-specifying the whole thing.